### ISSUE TITLE
Table On Hover mouse and also tab navigation styling

### ISSUE THREAD

**Issue by crchong1** [keepid/keepid_client#139]
On mouse hover should show the shading. Also, one should be able to cycle through the results using arrow keys and tab function for accessibility

**Comment by vanessa-hu** [comment]
on hover works via rowClasses. Not sure what tab nav means?
```
const rowClasses = (row, rowIndex) => {
            const { editRows, data } = this.state;
            var classes = 'table-row';
            if (editRows.has(row.id)) {
                classes += ' table-edit-row'; 
            }
            else if (data.length > 10 && rowIndex%2 === 0)
                classes += ' table-zebra';
                return classes;
        };
```
